src/ur/ur_driver.cpp

```diff
@@ -148,6 +154,8 @@ urcl::UrDriver::UrDriver(const std::string& robot_ip, const std::string& script_
   }
 
   reverse_interface_.reset(new control::ReverseInterface(reverse_port, handle_program_state));
+  // TODO swap to configurable or static port
```

Comment:
```suggestion
```
As is in the control script - could be left like this for now, but worth thinking about deciding if we want to change this now.

Reply:
See ca8fc29